src/utils/purge.js

```diff
@@ -24,3 +42,19 @@ module.exports = url => {
       throw e;
     });
 };
+
+/**
+ * Generic entrypoint for authenticated/unauthenticated purge reuqests.
+ * @param {string} url
+ * @returns {Promise<Object>} An object containing the request data
+ * @throws {Error} Throws an error if the request was not successful
+ */
+const purge = function (url) {
+  return (
```

Comment:
Not sure if these outer parenthesis are part of the Airbnb style, but they are definitely not needed

Reply:
That's the `python` part of the brain thinking.
I've updated the style here:
https://github.com/clay/cache-clear/pull/9/commits/94935a0ec81b117db90954a7d89dbe6de6353f11